### PR TITLE
Top_Menu_Made_Responsive_For_Mobile

### PR DESCRIPTION
**Pull request by gaurav-iitb**
please accept this pr with hacktoberfest-accepted label . if you liked it and please suggest any changes if you want
Thank You!

### CODE REVIEW

File: _sass/layout.scss [hunk 1]
```diff
@@ -27,7 +27,8 @@
 	}
 
 	.notMobile {
-		display: none !important;
+		// display: none !important;
```
Comment (TheLastProject) [inline _sass/layout.scss:30]: I don't like commented out code, if we need old code we can go back to older commits :)

```suggestion
```

File: _sass/layout.scss [hunk 2]
```diff
@@ -45,6 +46,7 @@
 	grid-template-columns: repeat(12, 1fr);
 	grid-template-rows: 115px 876px auto auto;
 	grid-column-gap: 30px;
+	overflow-x: hidden;  // changed here
```
Comment (TheLastProject) [inline _sass/layout.scss:49]: When I'm going to read the code later the common "changed here" won't tell me anything :)

```suggestion
	overflow-x: hidden;
```

File: Gemfile [hunk 3]
```diff
@@ -1,3 +1,9 @@
-ruby '~> 2.7'
+# frozen_string_literal: true
+
```
Comment (TheLastProject) [inline Gemfile:2]: More commented out code

```suggestion
```

File: Gemfile [hunk 4]
```diff
@@ -1,3 +1,9 @@
...
 source "https://rubygems.org"
-gem 'github-pages'
+
+# gem "rails"
```
Comment (TheLastProject) [inline Gemfile:5]: More commented out code

```suggestion
```

**Review by TheLastProject — CHANGES_REQUESTED** [review]
You made a lot of unrelated changes but given I'm not hosting the website on GitHub Pages currently and the new Gemfile looks more like the example on https://jekyllrb.com/docs/ruby-101/#gemfile this looks okay.

Please just remove the commented out code that I marked so it's a bit cleaner, then I can just merge this. It's not the drop-down I expected but it actually works really well so yeah, great, this is better than my idea :)

**Comment by TheLastProject** [comment]
Thanks!

File: Gemfile (hunk 4 above)
Comment (gaurav-iitb) [reply Gemfile]: thank you so much

**Pull request merged**